scripts/create_input.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import sys
+
+import predicting_the_past_import as import_data
+
+# user defined characteristics of cyclus simulation
+data_year = 2019
+start_year = 1965
+region = 'united_states'
+project = 'haleu'
+
+import_data.merge_coordinates('../database/Year-end Reactor Status_' + str(data_year) + '.csv',
+                              '../database/coordinates.sqlite', data_year)
+
+pris_file = '../database/reactors_pris_' + str(data_year) + '.csv'
+deployinst_tmpl = '../input/predicting-the-past/templates/' + \
+    region + '/deployinst_template.xml'
+inclusions_tmpl = '../input/predicting-the-past/templates/inclusions_template.xml'
+deployment_path = '../input/' + project + '/inputs/' + region + '/buildtimes'
+reactor_path = '../input/' + project + '/inputs/' + region + '/reactors'
+reactor_template = '../input/predicting-the-past/templates/reactors_template.xml'
+
+
+pris = import_data.import_csv(pris_file)
+
+
+recipes = import_data.import_csv('../database/vision_recipes/uox.csv', ',')
+recipe_template = import_data.load_template(
+    '../input/predicting-the-past/templates/recipes_template.xml')
+fresh33 = import_data.get_composition_fresh(recipes, 33)
+spent33 = import_data.get_composition_spent(recipes, 33)
+fresh51 = import_data.get_composition_fresh(recipes, 51)
+spent51 = import_data.get_composition_spent(recipes, 51)
+fresh100 = import_data.get_composition_fresh(recipes, 100)
+spent100 = import_data.get_composition_spent(recipes, 100)
+import_data.write_recipes(fresh33, spent33, recipe_template, 33, region)
+import_data.write_recipes(fresh51, spent51, recipe_template, 51, region)
+import_data.write_recipes(fresh51, spent51, recipe_template, 100, region)
```

Comment:
Can I recommend DRY:

```
burnups=[33, 55, 100]
for bu in burnups:
    fresh = import_data.get_composition_fresh(recipes, bu)
    spent = import_data.get_composition_spent(recipes, bu)
    import_data.write_recipes(fresh, spent, recipe_template, bu, region)
```

Reply:
Made this change

Reply:
Excellent. Resolving the conversation indicates that. When you've made all the changes that satisfy the review and are ready for a re-review, you can click the button next to my (or any reviewer's) name to re-request.